templates/cssskeleton.mustache

```diff
@@ -66,7 +72,12 @@
     {{/editmode}}
     {{/hidepaths}}
     {{^editmode}}
+    {{^showall}}
```

Comment:
Aren't lines 70-72 obsolete with this?

Reply:
No, this is needed to hide the paths in "normal" mode.

Reply:
Latest commit improved readability, I overlooked `hidepaths` condition at first, sry